.projenrc.js

```diff
@@ -47,7 +47,7 @@ const project = new JsiiProject({
   stability: 'stable',
   minNodeVersion: '10.17.0',
 
-  releaseBranches: ['master', '2.x', '10.x'],
+  releaseBranches: ['master', '2.x'],
```

Comment:
If projen "has issues with repositories that release from multiple branches, like this one", wouldn't we want to disable the 2.x branch as well? There hasn't been a change to that branch in 10 months, so it seems low-risk to also temporarily disable releasing it for now, just in case.

Reply:
Yes, I'll do that too. Good call.

Reply:
Done. I will submit a separate PR to disable this from `master` and `2.x`